rest-api/spring-rest/src/test/java/uk/gov/gchq/gaffer/rest/controller/OperationControllerTest.java

```diff
@@ -93,31 +88,30 @@ public void setUpController() {
         when(graphFactory.getGraph()).thenReturn(graph);
     }
 
-    @SuppressWarnings({"unchecked"})
     @Test
-    public void shouldReturnAllSupportedOperationsAsOperationDetails() {
-        // Given
-        when(store.getSupportedOperations()).thenReturn(Sets.newHashSet(AddElements.class, GetElements.class));
-
-        // When
-        final Set<OperationDetail> allOperationDetails = operationController.getAllOperationDetails();
-        final Set<String> allOperationDetailClasses = allOperationDetails.stream().map(OperationDetail::getName).collect(Collectors.toSet());
+    public void shouldReturnAllOperationsAsOperationDetailsInASortedManner() {
```

Comment:
`shouldReturnAllOperationsAsOperationDetailsInSortedOrder` might be clearer for the test name. Same for test below.